src/formMapper.js

```diff
@@ -0,0 +1,71 @@
+const formElSelector = 'form';
+const defaultElementSelectors = {
+    input: 'input',
+    checkbox: 'input[type="checkbox"]',
+    radio: 'input[type="radio"]',
+    select: 'select',
+    multiSelect: 'select[multiple]',
+    textArea: 'textarea'
```

Comment:
These 6 settings could be reduced to one "inputSelector" setting that would accept a selector string. The user of this module could then pass in as simple or as complex a selector as they desired.  Examples: 'input, select, textarea' (<- probably the default), '.js-input, .js-checkbox, .js-radio', etc.  This would also reduce some of the code in your getInputElements method. What you have isn't necessarily wrong, I just don't see a lot of benefit in splitting out the selector setting into multiple settings.